{{ cookiecutter.repo_name }}/buildSrc/src/main/java/plugins/AppCenterConfigPlugin.kt

```diff
@@ -0,0 +1,49 @@
+package plugins
+
+import com.betomorrow.gradle.appcenter.extensions.AppCenterExtension
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import utils.GitHelper
+import utils.propOrEmpty
+
+class AppCenterConfigPlugin : Plugin<Project> {
+    override fun apply(target: Project) {
+        if (System.getenv("CI") == "true") {
+            target.plugins.apply("com.betomorrow.appcenter")
+            val extension = target.extensions.getByType(AppCenterExtension::class.java)
+            try {
+                extension.configure(target)
+            } catch (e: Exception) {
+                e.printStackTrace()
+            }
+        }
+    }
+
+    private fun AppCenterExtension.configure(target: Project) {
+        val gitHelper = GitHelper(target.rootDir)
+
+        apiToken = target.propOrEmpty("APPCENTER_TOKEN")
+        ownerName = "RightpointApps"
```

Comment:
I'm not a huge fan to hard code this value, since the template can be used by anybody outside the org. Maybe a cookiecutter param ?

Reply:
Good point. I'll make that update.